inc/admin/ui/bulk.php

```diff
@@ -138,7 +138,7 @@ function _imagify_display_bulk_page() {
 								<?php
 								printf(
 									/* translators: you can use %s to include a line break. */
-									__( 'that\'s the number of images you optimized with Imagify', 'imagify' ),
+									__( 'that\'s the number of original images you optimized with Imagify', 'imagify' ),
```

Comment:
Not directly related to the fix, but in WP Rocket it was decided to not use abbreviation for things like that's, you're, etc…notably to prevent the backslash use.

Reply:
I also avoid them when possible.